### PR TITLE
Sumo 216336 - Beta doc for RBAC for indexes

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds a closed beta article:
https://help.sumologic.com/docs/manage/users-roles/roles/rbac-for-indexes

It also cleans up image and link coding in some other roles articles.

Issue number: [SUMO-216336](https://sumologic.atlassian.net/browse/SUMO-216336)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by jpipkin1** [comment]
This PR is in draft until all SME review comments are in.

**Comment by jpipkin1** [comment]
This PR is ready for review.

File: docs/manage/users-roles/roles/construct-search-filter-for-role.md [hunk 1]
```diff
@@ -38,20 +41,21 @@ The role filter above denies access to log data whose  `_sourceCategory` begins
 
 The examples above are simple: they involve a single role, and hence a single role filter. 
 
-Typically however, a Sumo user will have multiple roles. If a user has multiple roles, Sumo ORs the several role filters and prepends that expression to the user’s queries with an AND, as discussed in [Multiple role filters and filter precedence](construct-search-filter-for-role.md).
+Typically however, a Sumo user will have multiple roles. If a user has multiple roles, Sumo ORs the several role filters and prepends that expression to the user’s queries with an AND, as discussed in [Multiple role filters and filter precedence](#multiple-role-filters-and-filter-precedence).
```
Comment (kimsauce) [inline docs/manage/users-roles/roles/construct-search-filter-for-role.md:44]: I'd maybe format like this to make it clear you're talking about code

```suggestion
Typically however, a Sumo user will have multiple roles. If a user has multiple roles, Sumo `OR`s the several role filters and prepends that expression to the user’s queries with an `AND`, as discussed in [Multiple role filters and filter precedence](#multiple-role-filters-and-filter-precedence).
```

File: docs/manage/users-roles/roles/create-manage-roles.md [hunk 2]
```diff
@@ -20,28 +22,19 @@ To create a role:
...
-1. **Capabilities**. In this section, click the checkbox beside each capability you want to grant to users with this role. For information about what each capability enables, see [Role Capabilities](role-capabilities.md).
+1. **Search Filter**. Use this option to control what log data users with this role can access. A search filter for a role defines what log data a user with that role can access. You can define a search filter using keywords, wildcards, metadata fields, and logical operators. Sumo prepends the search filter to each query that a user with the role runs. The search filter is invisible to the user, but limits the log results that are returned. See [Construct a Search Filter for a Role](/docs/manage/users-roles/roles/construct-search-filter-for-role) for details and examples on constructing search filters for roles. The simple role filter below prevents users from viewing logs from Sources that include "billing" in their name. `!_source="billing"`
```
Comment (kimsauce) [inline docs/manage/users-roles/roles/create-manage-roles.md:28]:
```suggestion
1. **Search Filter**. Use this option to control what log data users with this role can access. A search filter for a role defines what log data a user with that role can access. You can define a search filter using keywords, wildcards, metadata fields, and logical operators. Sumo prepends the search filter to each query that a user with the role runs. The search filter is invisible to the user, but limits the log results that are returned. See [Construct a Search Filter for a Role](/docs/manage/users-roles/roles/construct-search-filter-for-role) for details and examples on constructing search filters for roles. The simple role filter below prevents users from viewing logs from Sources that include "billing" in their name (`!_source="billing"`).
```

File: docs/manage/users-roles/roles/rbac-for-indexes.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+An index based search filter allows or denies access to [search indexes](/docs/manage/partitions-data-tiers/). 
+
+1. [Create a role](/docs/manage/users-roles/roles/create-manage-roles#create-a-role).
+1. Under **Search Filter**, select **Index based**.
+1. Select one of the following:
+   * **All indexes**. Allow access to all indexes.
+   * **Allow few indexes**. Allow access to only the selected indexes. 
+   * **Deny few indexes**. Deny access to the selected indexes. 
+1. If you choose **Allow few indexes** or **Deny few indexes**, choose the indexes in the **Select Indexes** box that appears.<br/><img src={useBaseUrl('img/users-roles/index-based-filter.png')} alt="Index based filter" style={{border: '1px solid black'}} width="400"/>
```
Comment (kimsauce) [inline docs/manage/users-roles/roles/rbac-for-indexes.md:51]:
```suggestion
1. If you choose **Allow few indexes** or **Deny few indexes**, choose the indexes in the **Select Indexes** box that appear.<br/><img src={useBaseUrl('img/users-roles/index-based-filter.png')} alt="Index based filter" style={{border: '1px solid black'}} width="400"/>
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor suggestions

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**